### DISCUSSION TITLE
Why do we see a multiplicity of Negotiated Rate for the same medical service?

### DISCUSSION THREAD

**Discussion by dgolden** [CMSgov/price-transparency-guide#633] [Q&A]
Keyark has encountered the situation where diverse negotiated rates* for In-Network MRF cannot be explained by any contextual attributes provided by the MRF, including billing_class, billing_code_modifier, and service_code. We see this for several carriers/networks, for a significant fraction of their billing codes, and where the service_code array is populated. There is nothing written to additional_information. The example below is from querying an MRF published for October 2022:
  
![image](https://user-images.githubusercontent.com/6385204/200220888-7faf7471-6f4f-457e-8461-eb63aaacbf88.png)
 
We are handling this by reporting a minimum and maximum rate, but the situation is unsettling.

What's the likely explanation? Should this multiplicity in rates be considered as an acceptable level of precision in price transparency, or is this an issue that payers need to address? 
 
*Discussed in Answer #16 of https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification

**Comment by shaselton-usds** [comment]
Hi @dgolden 

It looks like this data has been modified from the original source, so it is a little hard to make a determination of the different negotiated rates. Maybe post a link to where the file is located in which you are modifying the data would be helpful to review?

**Reply by dgolden** [reply]
Howdy @shaselton-usds     Here are the details of the Aetna MRF we processed:

Tax ID: 370701328
NPI: 1780638478
TOC: 2022-10-05_97054907_index.json.gz
MRF URI (still accessible) 
https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICSI/2022-10-05/inNetworkRates/2022-10-05_d390517a-f6f5-4fbc-82dc-e84ab15f3aaf_Aetna-Life-Insurance-Company.json.gz

**Comment by joseph-kendall** [comment]
You can have a code in a contract more than once. It could be part of a bundle. It could be by itself. It could also be part of a default to capture anything that isn't set out in the contract specifically. Modifiers, POT (Place of Treatment or Service), etc. could allow the code to be included multiple times in a contract with different contract rates.

**Reply by medlyze** [reply]
Agree with the above.  Rates may seem duplicates but they are unique when including the site of service, modifiers, etc.

**Comment by AshleyOboe** [comment]
We've identified this issue in the Keywell dataset as well.  

Below is a GDrive is a link with about 1000 examples per payer (UHC, BCBS, Cigna, Aetna) where there are multiple rates but they have the same service code or set of service codes, modifier, negotiation type, etc.  The extract includes one code, 72148 (MRI Spinal Canal Lumbar w/o contrast material), in the DFW market.  The CSV files include the original filename and URL where the rates were published. 

The examples include only records that did not have a modifier included (so it should theoretically be the base imaging rate, whereas rates with modifiers would be technical reads only etc).  Also, the NPIs are all Type II (organizational).  

As an aside, In the BCBS files,  we don't see rates with modifiers for this bill code and many of the base rates look far too low for an MRI; my assumption is that these missing the modifiers (i.e. TC and 26 for this bill code).  

Many of the URLs will be expired but I have the original downloads anyone wants them to investigate the issue. 

Drive Link: https://drive.google.com/drive/folders/1zXHQ-eZSv2TkGPxcPmkf-UfmDvgZCP4p?usp=share_link

+ One example from each payer:

**UHC**
Tax ID: 453167434
NPI: 1194812495
Filename:2022-08-01_United-HealthCare-Services--Inc-_Insurer_EP1-50_C1_in-network-rates.json.gz 
Service Code: 11
URL: https://uhc-tic-mrf.azureedge.net/public-mrf/2022-08-01/2022-08-01_United-HealthCare-Services--Inc-_Insurer_EP1-50_C1_in-network-rates.json.gz
Reported Rates: $473.17, $431.94

**BCBS**
Tax ID: 815393966
NPI: 1598109597
Filename:2022-08_510_01C0_in-network-rates_12_of_29.json.gz
Service Code: 11, 81
URL(expired): https://bcbstx.mrf.bcbs.com/2022-08_510_01C0_in-network-rates_12_of_29.json.gz?&Expires=1663611777&Signature=gRUZE7i1c7qBo6N3kZ-APupGCBEzbjHKcAZWykTJC2J0Dr3qD-fUnWt~PRpZioZ~3ojHoqZrer6XypQ8SDEszlsY7hE4Lb9MaZUoHqtDp8ZnpsEMrVWE7rFD3VtqKFThM7v-e4JEH6-hzJ6zTvpbEj-OwkaVMSMsZxyH5RvQEa4NTadjl-DHtIodYEwWoqSs6Ji-w0wLZ3YemZTiNQPaZGMst5poNgllqvp7cA4xGdl-9jYd0Pql39c5LkHi5Naka9M3LSj3IcMwk5IAk7WxmfLWCUuoPQohGJN77BgMypvNCQguKHFiwAhFn7hDttrGTYngG9-SvSLmLm7o4wOhFw__&Key-Pair-Id=K27TQMT39R1C8A
Reported Rates: $221.86, $155.3,$510, $255, $178.5, $606.39, $659.43

**Aetna**
Tax ID: 135598093
NPI: 1306089925
Filename: 2022-09-05_2f5453db-33ce-4a89-9840-0526566849c8_Aetna-Life-Insurance-Company.json.gz
Service Code: All service codes listed
URL: https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICSI/2022-09-05/inNetworkRates/2022-09-05_2f5453db-33ce-4a89-9840-0526566849c8_Aetna-Life-Insurance-Company.json.gz
Reported Rates:  $1741, $216, $1525

**Cigna**
Tax ID: 264000683
NPI: 1285649863
Filename: 2022-10-01_cigna-health-life-insurance-company_national-oap_in-network-rates.json.gz
Service Code: All service codes listed
URL (expired): https://d25kgz5rikkq4n.cloudfront.net/cost_transparency/mrf/in-network-rates/reporting_month=2022-10/2022-10-01_cigna-health-life-insurance-company_national-oap_in-network-rates.json.gz?Expires=1667786649&Signature=BIIe4JLcpSQR~TC9MzdUmAdoK1GoLzlN5eGSB7xYTGtEGerJfMcstGQ8cHIB0y7Ixo5Zuw1X8X~SNyvY9yZeB0Q559IStgpDh3IFb4g84oUZZt4Zrns87zzkPRJ0vLxbXo3itBciklYakdfz3IQaY5idwWEo7VcpBe1hy26JqRy9ZZ8oxlLzd5NnCjNdtoMBtq2-AvNk-zzVh6~gLsxFm5lkKbfuAtQViMjGH7xO~-xfiIX3DpFJ0SfzF86v84MZggF-owiFjleU8wkMcDfIuLapwJTxbla7Mi4cdkgt7K2xKsaRtreypu2zuXY3ZJwWscx9zRap6PQ0HqU0jhK0Kw__&Key-Pair-Id=K1NVBEPVH9LWJP
Reported Rates:  $375, $464

**Comment by clovis517** [comment]
I am finding this issue as well, in some MRFs there are 10+ prices listed with zero differentiation across other code/price/rate/provider fields, rendering the MRF unusable.  In many of the schemas it is _implied_ that some of the fields constitute a unique key.  It would be helpful if this was explicitly stated.   

From this discussion https://github.com/CMSgov/price-transparency-guide/discussions/356#discussioncomment-2588977 

> I'd recommend [@rakeshkatte](https://github.com/rakeshkatte)'s approach. If there is a primary billing code, use that for the billing_code and billing_code_type and then populate additional_information with the other codes that are apart of the arrangement for context.  

I conclude that the additional_information can and should be used as the differentiator (part of the implied key).

@shaselton-usds , I would appreciate your thoughts on this:
Should we consider the first entry of the following MRF to be compliant with the mandate even though it does not provide any way to differentiate between the two rates?  There are other examples there are numerous values ranging from pennies to hundreds of dollars.
https://priorityhealthtransparencymrfs.s3.amazonaws.com/PPO.html
```
{
   "reporting_entity_name":"Priority Health",
   "reporting_entity_type":"health insurance issuer",
   "in_network":
      [
         {
            "negotiation_arrangement":"ffs",
            "name":"Exc Les Oral Vestibule Simple Rep",
            "billing_code_type":"CPT",
            "billing_code_type_version":"",
            "billing_code":"40812",
            "description":"Exc Les Oral Vestibule Simple Rep",
            "negotiated_rates":
               [
                  {
                     "negotiated_prices":
                        [
                           {
                              "service_code":["11","22"],
                              "negotiated_type":"fee schedule",
                              "negotiated_rate":312.54,
                              "expiration_date":"9999-12-31",
                              "billing_class":"professional"
                           },
                           {
                              "service_code":["11","22"],
                              "negotiated_type":"fee schedule",
                              "negotiated_rate":490.22,
                              "expiration_date":"9999-12-31",
                              "billing_class":"professional"
                           }
                        ],
                     "provider_groups":[{"npi":[1386922003],"tin":{"type":"ein","value":"380593405"}}]
                  },
                  {
                     "negotiated_prices":

```

**Reply by frishrash** [reply]
Any news on this issue? it really does render some of these published rates useless.
Is there a formal process to "dispute" such MRF files?

Thanks!

**Comment by Nabins** [comment]
Rates may look duplicates for specified procedure code but they differ from other fields such as negotitaed type,  Place of service(pos), billing code modifier,
billing class, npi(or sometimes the location of npi), tin

**Reply by quadrupole** [reply]
Even when you account for those dimensions, we are still seeing multiple rates as well. My guess is that the difference comes from plan ids or other factors that are specified in the index files related to the MRFs but not in the MRF itself? For example plans like "SELECT EPO" "SELECT PLUS" are specified in the index files, but you dont know which of them pertains to a particular rate, but this would account for the potential multiplicity.

**Comment by knash05** [comment]
We’ve identified several scenarios where rates are duplicated. For example, when a provider practices in multiple locations or when medical groups include multiple specialties within the same group—such as different rates for physicians, PAs, and NPs. Additionally, some contracts include tiered structures, like the first two days at a CASE rate followed by a PER DIEM arrangement.

These variations occur for many reasons within the current schema bounds, but additional elements are needed to properly support these use cases. While some have suggested workarounds—such as appending details to existing elements—these approaches risk creating inconsistencies, as plans could choose whether or not to adopt them.

A more sustainable solution would be to add these additional known elements to the schema, enabling users to clearly differentiate and maintain consistency.
